server/example/org.eclipse.glsp.ide.workflow.editor/.checkstyle

```diff
@@ -11,4 +11,14 @@
     <additional-data name="cache-file-location" value="EMF.cloud Checkstyle (8.44)_1638214295838_cache.xml"/>
     <additional-data name="cache-file" value="true"/>
   </local-check-config>
+  <local-check-config name="maven-checkstyle-plugin default" location="jar:file:/C:/Users/viskee/.m2/repository/org/eclipse/emfcloud/org.eclipse.emfcloud.checkstyle/0.1.0-SNAPSHOT/org.eclipse.emfcloud.checkstyle-0.1.0-SNAPSHOT.jar!/emfcloud-checkstyle-8.44.xml" type="remote" description="maven-checkstyle-plugin configuration default">
+    <property name="checkstyle.header.file" value="D:\dev\eclipse\eclipse-glsp-eclipse-integration\ws\.metadata\.plugins\org.eclipse.core.resources\.projects\org.eclipse.glsp.ide.workflow.editor\com.basistech.m2e.code.quality.checkstyleConfigurator\checkstyle-header-default.txt"/>
+    <property name="checkstyle.cache.file" value="${project_loc}/target/checkstyle-cachefile"/>
+  </local-check-config>
+  <fileset name="java-sources-default" enabled="true" check-config-name="maven-checkstyle-plugin default" local="true">
+    <file-match-pattern match-pattern="^src/.*\.java" include-pattern="true"/>
+    <file-match-pattern match-pattern="^src.*.*src-gen/.*\.java" include-pattern="false"/>
+    <file-match-pattern match-pattern="^src/main/resources.*\.properties" include-pattern="true"/>
+    <file-match-pattern match-pattern="^src/test/resources.*\.properties" include-pattern="true"/>
+  </fileset>
```

Comment:
I think this change is not really required. Also it contains absolute paths to your workspace.

Reply:
Thanks, I don't know how this happened, but I noticed the same thing when I looked into the template repository you linked earlier. I revert those and the other changes like this